src/qml/bitcoin.cpp

```diff
@@ -13,6 +13,7 @@
 #include <noui.h>
 #include <qml/imageprovider.h>
 #include <qml/nodemodel.h>
+#include <qml/appmode.h>
```

Comment:
For 872807a05daeae9752a39df275a3aff602c942e4, please make the following style changes:

``` diff
diff --git a/src/qml/appmode.h b/src/qml/appmode.h
index 0148a4240..e45977318 100644
--- a/src/qml/appmode.h
+++ b/src/qml/appmode.h
@@ -15,14 +15,12 @@ class AppMode : public QObject
     Q_PROPERTY(QString state READ state NOTIFY modeChanged)
 
 public:
-    enum Mode
-    {
+    enum Mode {
         DESKTOP,
         MOBILE
     };
 
-    explicit AppMode(Mode mode) :
-        m_mode(mode)
+    explicit AppMode(Mode mode) : m_mode(mode)
     {
     }
 
@@ -31,12 +29,12 @@ public:
     QString state()
     {
         switch (m_mode) {
-            case MOBILE:
-                return "MOBILE";
-            case DESKTOP:
-                return "DESKTOP";
-            default:
-                return "DESKTOP";
+        case MOBILE:
+            return "MOBILE";
+        case DESKTOP:
+            return "DESKTOP";
+        default:
+            return "DESKTOP";
         }
     }
     Mode mode() const { return m_mode; }
diff --git a/src/qml/bitcoin.cpp b/src/qml/bitcoin.cpp
index c8687b5ed..291b39f53 100644
--- a/src/qml/bitcoin.cpp
+++ b/src/qml/bitcoin.cpp
@@ -11,9 +11,9 @@
 #include <logging.h>
 #include <node/interface_ui.h>
 #include <noui.h>
+#include <qml/appmode.h>
 #include <qml/imageprovider.h>
 #include <qml/nodemodel.h>
-#include <qml/appmode.h>
 #include <qml/util.h>
 #include <qt/guiconstants.h>
 #include <qt/guiutil.h>
```

You can get the same diff locally by running the following command on 872807a05daeae9752a39df275a3aff602c942e4
```
git diff -U0 HEAD~1.. | ./contrib/devtools/clang-format-diff.py -p1 -i -v
```

Reply:
Thanks, that's really good info on the cpp style too.